package.json

```diff
@@ -36,6 +33,7 @@
   "dependencies": {
     "@types/json-schema": "^7.0.9",
     "js-yaml": "^3.14.1",
+    "prettier": "2.0.5",
```

Comment:
Shouldn't this be a peer dependency? The initial idea was to not include `prettier` in the bundle at all. It will work automatically based on whether `prettier` is imported or not. (But it wasn't implemented correctly.)

Reply:
I think most builds will just break if `prettier` is missing. I think it’s better to include it as a dependency. I understand if one wants to exclude it from the bundle because of its size, but it’s usually no big deal if this dependency is in `node_modules`.

Also when I tried earlier, newer versions of `prettier` didn’t work. If this isn’t included as a regular dependency, this will most likely cause hard to troubleshoot issues, especially because many users already use a newer version of `prettier` to format their own code.

I do think maybe we can revisit this later.

Reply:
OK yeah it makes sense then.